src/oemof/tabular/facades/experimental/battery_electric_vehicle.py

```diff
@@ -453,3 +453,259 @@ def build_solph_components(self):
+    drive_consumption : array-like
+        Profile of drive consumption of the fleet (relative to capacity).
+    max_charging_power : int
+        Max charging/discharging power of all vehicles (e.g. in MW)
+    availability : array-like
+        Ratio of available capacity for charging/vehicle-to-grid due to
+        grid connection.
+    efficiency_charging: float
+        Efficiency of charging the batteries, default: 1
+    v2g: bool
+        If True, vehicle-to-grid is enabled, default: False
+    loss_rate: float
+    min_storage_level : array-like
+        Profile of minimum storage level (min SOC)
+    max_storage_level : array-like
+        Profile of maximum storage level (max SOC).
+    balanced : boolean
+        Couple storage level of first and last time step.
+        (Total inflow and total outflow are balanced.)
+    transport_commodity: None
+        Bus for the transport commodity
+    input_parameters: dict
+        Dictionary to specify parameters on the input edge. You can use
+        all keys that are available for the  oemof.solph.network.Flow class.
+    output_parameters: dict
+        see: input_parameters
+    """
+
+    # TODO: match data formats with actual data
+
+    label: str = "ind_mob_sec"
+
+    electricity_bus: Bus
+
+    # transport_commodity_bus: Bus = None Todo: nötig? wird unten angelegt
+
+    charging_power_g2v: int
+
+    charging_power_v2g: int
+
+    charging_power_inflex: int
+
+    charging_potential: int
+
+    availability: Sequence[float]
+
+    storage_capacity_g2v: int
+
+    storage_capacity_v2g: int
+
+    storage_capacity_inflex: int
+
+    # initial_storage_capacity=, ToDo: weglassen?
+
+    min_storage_level: Sequence[float]
+
+    max_storage_level: Sequence[float]
+
+    # drive_power: int todo: s.u.
+
+    drive_consumption: Sequence = None
+
+    v2g: bool = (False,)  # todo: hier notwendig? wird unten hard-gecoded
+
+    loss_rate: float = 0
+
+    efficiency_mob_g2v: float = 1
+
+    efficiency_mob_v2g: float = 1
+
+    efficiency_sto_in: float = 1
+
+    efficiency_sto_out: float = 1
+
+    efficiency_mob_electrical: float  # = 1 todo: unit?
+
+    pkm_conversion_rate: float  # todo: unit?
+
+    # expandable: bool = False todo: ?
+
+    lifetime: float
+
+    # age=, todo: ?
+
+    # invest_c_rate=, todo: ?
+
+    bev_storage_capacity: int
+
+    # bev_capacity=, todo:?
+
+    bev_invest_cost: int
+
+    fixed_costs: int
+
+    variable_costs: int
+
+    # balanced=, todo:?
+
+    input_parameters_inflex: dict = field(default_factory=dict)
+
+    # output_parameters: dict = field(default_factory=dict), todo: weglassen?
+
+    # max_charging_power: Union[float, Sequence[float]] todo: s.u.
+
+    def build_solph_components(self):
+        transport_commodity_bus = Bus(label="transport_commodity")
+        transport_commodity_bus.type = "bus"
+
+        mobility_nodes = [transport_commodity_bus]
+
+        bev_controlled_g2v = Bev(
+            label="G2V",
```

Comment:
label needs to be a combination of meta-facade label + technology postfix: e.g. 'bev_mcar' + "_G2V"